doc/release/0.16.0-notes.rst

```diff
@@ -79,6 +98,15 @@ Tukey window was added as `scipy.signal.tukey`.
 The function for computing digital filter group delay was added as
 `scipy.signal.group_delay`.
 
+`scipy.sparse` improvements
+---------------------------
+
+The function `scipy.sparse.norm`, which computes sparse matrix norms, was
+added.
+
+The function `scipy.sparse.random`, which generates sparse matrices with
+normally distributed random samples, was added.
```

Comment:
Sorry to pester --- "... which allows to draw random variates from an arbitrary distribution". The scope of that PR has changed a bit en route.